python/GafferArnoldUI/ArnoldOptionsUI.py

```diff
@@ -41,6 +41,15 @@
 import GafferUI
 import GafferArnold
 
+def __renderingSummary( plug ) :
+
+	info = []
+	if plug["bucketSize"]["enabled"].getValue() :
+		info.append( "bucketSize %d" % plug["bucketSize"]["value"].getValue() )
```

Comment:
Use capitals and spaces for the UI text. Same below, including capitalising the string value.